doc/ref_arch/openstack/chapters/chapter03.rst

```diff
@@ -97,7 +97,7 @@ enable workloads to be logically separated from each other with:
 
 -  differentiated set of associated users
 -  role-based access of two levels – admin or member (see `RBAC security
-   section <./chapter06.md#6314-rbac>`__).
+   section <./chapter06.md#rbac>`__).
```

Comment:
post ch06 rst conversion
```suggestion
   section <./chapter06#rbac>`__).
```

Reply:
@pgoyal01 It may appear too cautious, but I prefer we remove the .md at the end when the all rst files will be cleaned up.